### PR TITLE
Fix Open Graph Info

### PR DESCRIPTION
**Pull request by bryon-czoch**

### CODE REVIEW

**Comment by bryon-czoch** [comment]
...yeah cant login to that meh

File: src/components/headMetaTags.js [hunk 1]
```diff
@@ -44,10 +47,16 @@ class HeadMetaTags extends Component {
...
+          {property: 'og:title', content: {title}},
+          {property: 'og:image', content: {logo}},
+          {property: 'og:description', content: {description}},
+          {property: 'og:url', content: {url}}
         ]}
         script={scripts}
         title={title}
+        logo={logo}
+        description={description}
+        url={url}
```
Comment (paaatrick) [inline src/components/headMetaTags.js]: I don't believe these are valid properties on the `Helmet` component:

https://github.com/nfl/react-helmet/blob/release/5.0.0/src/Helmet.js#L30

File: package.json [hunk 2]
```diff
@@ -68,7 +68,7 @@
     "@babel/register": "7.0.0",
     "babel-loader": "^8.0.2",
     "babel-preset-react-app": "7.0.1",
-    "cross-env": "^5.2.0",
+    "cross-env": "latest",
```
Comment (paaatrick) [inline package.json]: I don't think we want to upgrade this at this time

**Comment by paaatrick** [comment]
Closing due to inactivity

**Comment by bryon-czoch** [comment]
hm yeah not sure wth i was thinking, might've been hyped up on TechLead

**Pull request closed without merging**